app/javascript/components/patient/close_contacts/CloseContactTable.js

```diff
@@ -0,0 +1,485 @@
+                  </Button>
+                </Col>
+                <Col xl={6} lg={10} md={12}>
+                  <InputGroup size="md" className="mt-3 mt-md-0 ">
+                    <InputGroup.Prepend>
+                      <OverlayTrigger overlay={<Tooltip>Search by First Name, Last Name, Phone Number, Email, Assigned User, or Contact Attepts.</Tooltip>}>
+                        <InputGroup.Text className="rounded-0">
+                          <i className="fas fa-search"></i>
+                          <label
+                            htmlFor="close-contact-search-input"
+                            className="ml-2 mb-0"
+                            aria-label="Search Close Contact Table by First Name, Last Name, Phone Number, Email, Assigned User, or Contact Attepts.">
+                            Search
+                          </label>
+                        </InputGroup.Text>
+                      </OverlayTrigger>
+                    </InputGroup.Prepend>
+                    <Form.Control
+                      id="close-contact-search-input"
+                      autoComplete="off"
+                      size="md"
+                      name="search"
+                      onChange={this.handleSearchChange}
+                      aria-label="Search"
+                    />
+                  </InputGroup>
+                </Col>
+              </Row>
+              <div className="mb-4">
+                <CustomTable
+                  dataType="close-contacts"
+                  columnData={this.state.table.colData}
+                  rowData={this.state.table.rowData}
+                  totalRows={this.state.table.totalRows}
+                  handleTableUpdate={query => this.updateTable({ ...this.state.query, order: query.orderBy, page: query.page, direction: query.sortDirection })}
+                  handleEntriesChange={this.handleEntriesChange}
+                  isLoading={this.state.isLoading}
+                  page={this.state.query.page}
+                  handlePageUpdate={this.handlePageUpdate}
+                  entryOptions={this.state.entryOptions}
+                  entries={this.state.query.entries}
+                  tableCustomClass="table-has-dropdown"
+                />
+              </div>
+            </div>
+          </Card.Body>
+        </Card>
+        {this.state.showDeleteModal && (
+          <DeleteDialog
+            type={'Close Contact'}
+            delete={this.handleDeleteSubmit}
+            toggle={this.toggleDeleteModal}
+            onChange={this.handleDeleteUpdate}
+            show_text_input={true}
+          />
+        )}
+        {(this.state.showAddModal || this.state.showEditModal) && (
+          <CloseContactModal
+            title={this.state.showAddModal ? 'Add New Close Contact' : 'Edit Close Contact'}
+            currentCloseContact={this.state.showAddModal ? {} : this.getCurrentCloseContact()}
+            onClose={this.toggleAddModal}
```

Comment:
Bug here- you can't cancel the edit modal, because this only toggles the add modal. Symptom is if you cancel the edit modal, it just changes the title of the modal to "Add..."

Reply:
Good catch! Fixed.